CHANGES.rst

```diff
@@ -49,6 +49,8 @@ Specviz
 Other Changes and Additions
 ---------------------------
 
+- Change default collapse function to sum. [#1229]
```

Comment:
If you insist the entry should stay here, I guess that's alright. No one reads this anyway, right? Should clarify the entry a little though.

```suggestion
- Change default collapse function to sum.
  This affects collapsed spectrum in Cubeviz and its Collapse plugin default. [#1229]
```

Reply:
Sorry, when I replied to @kecnry comment I did not see your comments (the page had not updated). I think this is a good clarification, thank you!